src/components/CodeInput.tsx

```diff
@@ -123,7 +123,7 @@ export default function CodeInput({
 
             {showInput ? (
               <TextInput
-                textContentType={shortVerificationCodesEnabled ? 'username' : undefined}
+                textContentType={shortVerificationCodesEnabled ? 'oneTimeCode' : undefined}
```

Comment:
hope this doesn't cause problems, textContentType affects iOS only and "oneTimeCode" allows us to automatically pick up the value from received messages